ppl/archive/schema.go

```diff
@@ -95,6 +96,10 @@ type Archive struct {
 	DataOrder        zbuf.Order
 	LogSizeThreshold int64
 	LogFilter        []ksuid.KSUID
+
+	immfiles interface {
```

Comment:
How about just type `immcache.ImmutableCache`, as in `OpenOptions.ImmutableCache`?

Reply:
I'm hesitant about this because this is not necessarily a cache- caching could be disabled, the file could not exist in the cache and need to be fetched.